src/components/table/examples/table-layout-fitDataFill.tsx

```diff
@@ -0,0 +1,69 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { invoices, Invoice } from './invoices';
+
+/**
+ * Layout
+ * Columns and their content can be decisive factors in how a table is
+ * preferred to rendered in the user interface. To set your preferred
+ * rendering, choose one of the available `layout` properties.
+ *
+ *
+ * ````tsx
+ * layout="fitDataFill"
```

Comment:
> * layout="fitDataFill"

I found the naming of the layouts not that self-explanatory 
What do you think about these suggestions: 
<img width="846" alt="Screenshot 2022-09-06 at 11 02 32" src="https://user-images.githubusercontent.com/50618208/188594004-276183d4-571f-4093-913c-143b150786e6.png">
<img width="844" alt="Screenshot 2022-09-06 at 11 02 48" src="https://user-images.githubusercontent.com/50618208/188594060-98141747-6666-481d-a913-a2802781d3ca.png">
<img width="864" alt="Screenshot 2022-09-06 at 11 03 13" src="https://user-images.githubusercontent.com/50618208/188594144-fd7d1558-3086-4926-bb30-7bd255314257.png">

<img width="843" alt="Screenshot 2022-09-06 at 11 03 23" src="https://user-images.githubusercontent.com/50618208/188594197-67f5aa7e-630c-40b6-a231-792cdbc39225.png">

Reply:
Yes, naming that Tabulator has chosen are a bit weird. I think they try to explain the table behavior, while we try to explain the column behavior. :D

The question is whether we can we do so or not: 
if user choses `responsiveColumns` we convert it to `fitColumns` and pass it to the table? 
if the choose `lowDensity` we convert it to `fitData` and pass it to the table.

I think it should be easy. But I don't know how to do it.

Reply:
Yes, naming that Tabulator has chosen are a bit weird. I think they try to explain the table behavior, while we try to explain the column behavior. :D

The question is whether we can we do so or not: 
if user choses `responsiveColumns` we convert it to `fitColumns` and pass it to the table? 
if the choose `lowDensity` we convert it to `fitData` and pass it to the table.

I think it should be easy. But I don't know how to do it. Do you know how to do it?